ui/ducks/send/send.js

```diff
@@ -1641,6 +1755,36 @@ const slice = createSlice({
         slice.caseReducers.validateGasField(state);
         slice.caseReducers.validateSendState(state);
       })
+      .addCase(fetchSwapAndSendQuotes.pending, (state) => {
+        const draftTransaction =
+          state.draftTransactions[state.currentTransactionUUID];
+
+        if (draftTransaction) {
+          draftTransaction.quotes = draftTransactionInitialState.quotes;
+          draftTransaction.swapQuotesError = null;
+          draftTransaction.isSwapQuoteLoading = true;
+        }
+      })
+      .addCase(fetchSwapAndSendQuotes.fulfilled, (state, action) => {
+        const draftTransaction =
+          state.draftTransactions[state.currentTransactionUUID];
+
+        if (draftTransaction) {
+          draftTransaction.isSwapQuoteLoading = false;
+          if (action.payload) {
+            draftTransaction.quotes = action.payload.quotes;
+          }
+        }
+      })
+      .addCase(fetchSwapAndSendQuotes.rejected, (state, action) => {
+        const draftTransaction =
+          state.draftTransactions[state.currentTransactionUUID];
+
+        if (draftTransaction) {
+          draftTransaction.isSwapQuoteLoading = false;
+          draftTransaction.swapQuotesError = action.error.message;
```

Comment:
Is this error going to be displayed?

Reply:
Yep, but it gets caught and converted to `SWAPS_QUOTES_ERROR`.

<img width="412" alt="Screenshot 2024-04-15 at 1 05 31 PM" src="https://github.com/MetaMask/metamask-extension/assets/44588480/96980218-3c64-40f7-b526-0922211d20a0">